src/capital-project/capital-project.repository.ts

```diff
@@ -81,6 +110,13 @@ export class CapitalProjectRepository {
             eq(capitalProject.id, capitalCommitment.capitalProjectId),
           ),
         )
+        .leftJoin(
```

Comment:
I hadn't considered joining on the common table expression, rather than selecting from it. Nice